docs/release/README.md

```diff
@@ -75,7 +75,10 @@ Follow these steps to cut a new Katib release:
    git clone git@github.com:kubeflow/katib.git $GOPATH/src/github.com/kubeflow/katib
    ```
 
-1. Make sure that you can build all Katib images:
+1. Make sure that you can build all Katib images. **Note** that
+   your Docker Desktop should
+   [enable containerd image store](https://docs.docker.com/desktop/containerd/#enable-the-containerd-image-store)
+   to build multi-arch images:
```

Comment:
Could you update the developer docs as well?
https://github.com/kubeflow/katib/blob/master/docs/developer-guide.md#requirements

Also, could you update the minimum required Docker version with v24.0.0 since the `containerd image store` is introduced in that version: https://github.com/moby/moby/releases/tag/v24.0.0

Reply:
@tenzen-y Makes sense, I updated the Developer guide.